### ISSUE TITLE
unable to pin ontap version 

### ISSUE THREAD

**Issue by hmanupati** [NetApp/terraform-provider-netapp-cloudmanager#148]
we are trying to deploy ontap version 9.11.1P3 by adding below variables to the terraform config and getting the attached screenshot 

module version: 
```
      source  = "netapp/netapp-cloudmanager"
      version = ">= 22.2.1"
```

variables used: 
```
  use_latest_version    = "false"
  ontap_version         = "9.11.1P3"
```

Error: 

```
Error: code: 400, message: {"message":"InvalidMetadataException: There is no valid configuration for Ontap Version 9.11.1P3, license ha-capacity-paygo, instance type n2-standard-32, region asia-northeast2.","causeMessage":"InvalidMetadataException: There is no valid configuration for Ontap Version 9.11.1P3, license ha-capacity-paygo, instance type n2-standard-32, region asia-northeast2."}
with module.netapp_asne1.module.systems["dr"].netapp-cloudmanager_cvo_gcp.this
on .terraform/modules/netapp_asne1/modules/system/cvo.tf line 1, in resource "netapp-cloudmanager_cvo_gcp" "this":
```

Please could you help with this. 

![Screenshot 2023-01-25 at 17 54 04](https://user-images.githubusercontent.com/37581158/214646263-6d504c60-6c38-4109-9f4a-bf47c2af2240.png)
![Screenshot 2023-01-25 at 17 59 28](https://user-images.githubusercontent.com/37581158/214646264-8b4e8171-ec5d-4c10-a8ed-a1097e48e807.png)

**Comment by lonico** [comment]
I don't think you can just use "9.11.1P3".
I can't find a reference, but I remember version identifiers were more complex.   You would need to check what is available in your region.

**Comment by lonico** [comment]
Here is an example:
![image](https://user-images.githubusercontent.com/5272858/214721149-dd4d5c8a-3102-403f-95c4-9a9d27185ce0.png)

**Comment by bryanheo** [comment]
@lonico Thank you for your update.
Could you let us know how to check what is available in our region? We are deploying the CVO in asia-northeast1 in GCP and the CVO is HA. Is it something we can find in GCP or Cloud Manager?

**Comment by lonico** [comment]
Is there a reason you cannot use the latest version?

I'm told it can take one or two weeks before a version reaches the cloud.   But I could not find how to get a list of versions.  You would need to contact GCP support or CloudManager (now BlueXP) support for that.

**Comment by bryanheo** [comment]
@lonico Thank you for your message.
We recently experienced an issue with the latest version option. We deployed CVO with the latest version option and the version was 9.11.1 and there were Cloud Storage issue (KTLS) causing the NFS access issue so we have made decision not to use the latest version before we test in nonprod environment. If you cannot find the list of the version (Permutation query), I will ask CloudManager support team as you mentioned. (I am not sure whether they are aware of this or not)
9.11.1P3 version has been released last December. You cannot see the version yet?
